soteria/lib/terminal/cli.ml

```diff
@@ -0,0 +1,15 @@
+let process_args no_color compact : Config.t = { no_color; compact }
+
+let term =
+  let no_color =
+    let doc = "Disable colored output" in
+    let env = Cmdliner.Cmd.Env.info ~doc "NO_COLOR" in
+    Cmdliner.Arg.(value & flag & info [ "no-color" ] ~docv:"" ~doc ~env)
+  in
+  let compact =
+    Cmdliner.Arg.(
+      value
```

Comment:
I try to have every optional argument as env variables, it has been very handy in the past.
I'd suggets `SOTERIA_COMPACT_DIAGNOSTICS`